examples/python/README.md

```diff
@@ -0,0 +1,15 @@
+## Support LLava-Interleave-Qwen-7B-hf
```

Comment:
Could you move it to `docs/en/multi_modal/llava_qwen.md`? `docs/en/qwen2_vl.md` can be a reference.